src/activation.jl

```diff
@@ -136,6 +166,19 @@ function celu(x::Real, α::Real = one(x))
     return ifelse(x ≥ 0, x / one(x), α * (exp(x/α) - one(x)))
 end 
 
+
+"""
+    trelu(x, theta = 1.0) = x > theta ? x : 0 
+
+Threshold Gated Rectified Linear   
+See [ThresholdRelu](https://arxiv.org/pdf/1402.3337.pdf)
+"""
+function trelu(x::Real,theta = one(x)) 
+    return ifelse(x> theta, x/ one(x), zero(x))
+end 
```

Comment:
Since we don't need to force floating point outputs, we can write just as
```julia
trelu(x::Real, theta=one(x)) = ifelse(x > theta, x, zero(x))
```